### DISCUSSION TITLE
Adaptations pour la Belgique

### DISCUSSION THREAD

**Discussion by phfroidmont** [elefan-grenoble/gestion-compte#757] [Ideas]
Avant toute chose, merci pour tout votre travail.

J'aide en ce moment à la création d'une coopérative en Belgique (https://www.lecellierdubaudet.be/projets/) et nous aimerions utiliser votre application. De petites adaptations sont nécessaires pour nos besoins et je peux ouvrir des PR si elles vous semblent pertinentes.

- Ajout d'un paramètre pour désactiver le contrôle d'accès par IP à la création d'un adhérent. Nous avons des IP dynamiques en Belgique, cette vérification n'est donc pas applicable.
- Autoriser les codes postaux à 4 ou 5 chiffres (au lieu de 5 exclusivement)

**Comment by raphodn** [comment]
Salut !

Merci de ton intérêt pour l'application "gestion-compte", et pour ta première PR (#756) :tada: 

Pas de soucis pour faire des PR, surtout si ca va dans le sens d'une démocratisation de l'outil :) 
Je t'ai mis en contributeur du projet, tu ne seras plus obligé de forker le projet, tu peux maintenant faire des branch directement depuis le repo.

En terme de workflow : idéalement créé une issue pour chacun de tes besoins. Et fait 1 ou plusieurs PR par issue. Comme ça on pourra échanger sur le besoin dans l'issue, et le code dans la PR.

Tu maitrises bien php symfony ? Je demande car je viens plutôt de python, j'ai dû apprendre au fil de l'eau pour aider à maintenir et faire avancer ce projet, on est preneur d'aide pour certaines tâches [dette technique](https://github.com/elefan-grenoble/gestion-compte/issues?q=is%3Aopen+is%3Aissue+label%3A"Dette+technique") si jamais :pray: 

Merci !
Raphael

**Comment by phfroidmont** [comment]
Salut !

Super, je vais créer les issues alors.
Je viens du monde Scala/Java donc je n'ai jamais utilisé Symfony, ceci étant dit, je veux bien donner un coup de main si on peut me guider un peu.
Je peux aider aussi pour la partie "DevOps" (comme on dit de nos jours). Notamment, je vais regarder pour mettre en place un build et un déploiement robuste parce que je ne suis pas à l'aise avec l'idée de faire un `git pull` en prod et d'avoir les dépendances de dev sur le serveur.
